test/unit/lib/classes/plugin-manager.test.js

```diff
@@ -432,19 +431,19 @@ describe('PluginManager', () => {
     });
 
     it('should create an empty cliOptions object', () => {
-      expect(pluginManager.cliOptions).to.deep.equal({});
+      expect(pluginManager.cliOptions).to.be.an('object').empty;
```

Comment:
Let's leave original form `to.deep.equal({})` is totally acceptable.

Same in below cases